### DISCUSSION TITLE
「好きな東方Projectキャラ？」 Do you have favorite Touhou Project character?

### DISCUSSION THREAD

**Discussion by aru1702** [NAIST-SE/AttractingContributors#407] [General]
I'm wondering as part of fan culture in Japan, Touhou Project is one of the famous well-known name.
But lately, I found this began its unpopularity due to many new comers gains their popularity.

Anyway, I'm trying to collect some ideas.
I found Touhou Project still gaining popularity based on fanbase (doujin) games, and merchandises.
Especially, if you know about Fumofumo series, from Gift company, designed by circle named ANGELTYPE.

I want you to say your favorite character from this series!
If you don't have any idea, I will post a picture about my friend collection, and you can ask me who is that.
If you don't know or very new to this, I welcome to any discussion and comments.

You can add directly and silently your favorites to my repo below:
Link to my repo: **https://github.com/aru1702/AttractingContributors-TouhouFumo**

Please keep this discussion healthy and without drama~

============================================================================

日本語はあまりできないですが、ここには喋って頑張ります。

一部の日本的なファンカルチャー、東方Projectは有名な一つと思います。
ですが、現在には色んなアニメとかマンガがもっと有名上げてのから東方Projectの好評のは減少されてになりました。

ところで、色々な考えもの僕を集めています。
同人ゲームと商品まだいっぱいなので、東方Projectの好評はまだ上げられるのを見つけました。
特にはGift製造のANGELTYPEサークルの名前からのふもふもシーリズのぬいぐるみです。

このシーリズ好きなキャラは教えてください。
考えものが無したら下のは友達のコレクション写真を見せてあげます。
もし何も知らないとか不慣れる方はここには何でも話し合ってことがぜひ大丈夫です。

直接して静かに君の好きなキャラを追加するOKです。
レポココ：**https://github.com/aru1702/AttractingContributors-TouhouFumo**

元気な話しあうのと話ドラマなしのをよろしく申し上げます。

**Comment by aru1702** [comment]
These are the examples from https://twitter.com/ebichan_puppet/status/1527877446061522945 and https://twitter.com/Mighty_Eldritch/status/1477142070280089603

![Mighty_Eldritch](https://github.com/aru1702/AttractingContributors-TouhouFumo/blob/main/pictures/others/FH_c4dLVUAARE5B.jpeg?raw=true)
![ebichan_puppet](https://github.com/aru1702/AttractingContributors-TouhouFumo/blob/main/pictures/others/FTQcb-RUYAAY9p4.jpg?raw=true)

**Comment by AtropsCooper** [comment]
My favorite added.
Help me please if you'd like to 🥺
https://github.com/AtropsCooper/AttractingContributors-Random

**Reply by aru1702** [reply]
I can see you added Koishi, she's also my favorite 😄 
I will look into your repo later, thank you for the contributions!

**Comment by asahi-kouryo** [comment]
I have added with mine

**Comment by Kotjik** [comment]
Hey, I contributed to your repo - would you mind adding something to mine as well? 🙂

**Reply by aru1702** [reply]
Yeah sure!
I think I already see yours and make a pull request 👍

**Comment by XC1996114514** [comment]
There are so many beautiful Touhou Project Dolls. Is them all yours?

**Reply by aru1702** [reply]
No, they are from friends in our internal Touhou forum
I only have some which I put the photo on my repo
I attached the twitter link above as sources, you can check their profile 👍
